backscatter/fitacf/determinations.py

```diff
@@ -15,15 +15,21 @@
 FITACF_REVISON_MINOR = int(config.get("fitacf","fitacf_revision_minor"))
 
 class Determinations(object):
-    """This is a class to construct a new dictionary of final deteminations
+    """This is a class to construct a new dictionary of final determinations
 
     This class holds the methods to convert fitted data into the final measurements
     of the plasma
 
     """
 
-    def __init__(self,raw_data,range_list,noise_pwr):
+    def __init__(self,raw_data,range_list,noise_pwr, tdiff):
         hdw_info = hdw[raw_data['stid']]
+
+        if tdiff is None:
+            self.tdiff = hdw_info['tdiff_a']        # TODO: Can we incorporate tdiff_b somehow?
```

Comment:
Good question. I think this is tricky as we would need to know which radars use it - and then parse the filename for 'a' or 'b' in the channel designator. Maybe...